packages/map/pages/_document.js

```diff
@@ -1,296 +1,26 @@
 import Document, {Html, Head, Main, NextScript} from "next/document";
+import fs from "fs";
+import path from "path";
 
+// read and place html files generated by publicgoods-scripts/packages/automation/index.js
+const head = fs.readFileSync(path.join(process.cwd(), "public/head.html"), "utf8");
+const navbar = fs.readFileSync(path.join(process.cwd(), "public/navbar.html"), "utf8");
+const footer = fs.readFileSync(path.join(process.cwd(), "public/footer.html"), "utf8");
+const scripts = fs.readFileSync(path.join(process.cwd(), "public/scripts.html"), "utf8");
+const templateClassName = fs.readFileSync(path.join(process.cwd(), "public/templateClassName.txt"), "utf8");
 class MyDocument extends Document {
   render() {
     return (
       <Html>
         <Head>
-          <title> &raquo; DPG Map</title>
-          <link
-            rel="icon"
-            type="image/png"
-            href="/wp-content/themes/dpga/favicon.png?v1622233620"
-          />
-          <meta name="robots" content="max-image-preview:large" />
-          <link
-            rel="stylesheet"
-            id="wp-block-library-css"
-            href="/wp-includes/css/dist/block-library/style.min.css"
-            type="text/css"
-            media="all"
-          />
-          <link
-            rel="stylesheet"
-            id="dpga-style-css"
-            href="/wp-content/themes/dpga/style.css"
-            type="text/css"
-            media="all"
-          />
-          <link
-            rel="stylesheet"
-            id="dpga-fontello-css"
-            href="/wp-content/themes/dpga/fontello/css/fontello.css"
-            type="text/css"
-            media="all"
-          />
-          <link
-            rel="stylesheet"
-            id="dpga-style-main-css"
-            href="/wp-content/themes/dpga/css/style.css"
-            type="text/css"
-            media="all"
-          />
-          <link rel="canonical" href="/map/" />
-          <script
-            dangerouslySetInnerHTML={{
-              __html: `var _gaq = _gaq || [];
-   _gaq.push(['_setAccount', 'UA-143534016-1']);
-   _gaq.push(['_trackPageview']);
-   (function() {
-   var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-   ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-   var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-   })();`,
-            }}
-          />
+          <html dangerouslySetInnerHTML={{__html: head}} />
```

Comment:
This is how we place parts of html files from `publicgoods-website/map/index.html` before rendering nextJS app. [Reference](https://stackoverflow.com/a/60362264)